### PR TITLE
Move `preloadedAudios`/`preloadedImages` inside `Browser`. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/library_browser.js [hunk 1]
```diff
@@ -881,7 +882,7 @@ var LibraryBrowser = {
   $getPreloadedImageData: (path, w, h) => {
     path = PATH_FS.resolve(path);
 
-    var canvas = /** @type {HTMLCanvasElement} */(preloadedImages[path]);
+    var canvas = /** @type {HTMLCanvasElement} */(Browser.preloadedImages[path]);
```
Comment (brendandahl) [inline src/library_browser.js:885]: Missing dependency on $Browser?

File: src/library_sdl.js [hunk 2]
```diff
@@ -2209,7 +2209,7 @@ var LibrarySDL = {
 
       if (!raw) {
         filename = PATH_FS.resolve(filename);
-        raw = preloadedImages[filename];
+        raw = Browser.preloadedImages[filename];
```
Comment (brendandahl) [inline src/library_sdl.js:2212]: Missing dependency on $Browser? (and the other spots in this file)

File: src/library_browser.js (hunk 1 above)
Comment (sbc100) [reply src/library_browser.js:885]: All symbols in this file implicitly depend on `$Browser` thanks to the `autoAddDeps(LibraryBrowser, '$Browser');` line at the bottom.

File: src/library_sdl.js (hunk 2 above)
Comment (sbc100) [reply src/library_sdl.js:2212]: This is also handled transitively I think sine we have `autoAddDeps(LibrarySDL, '$SDL');` below and SDL depends on `Browser.`   Maybe I should make this one explicit though.

**Review by brendandahl — APPROVED** [review]

**Pull request merged**